sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/TextAnalyticsAsyncClientTest.recognizeLinkedEntitiesForBatchInputShowStatistics[1].json

```diff
@@ -1,26 +1,26 @@
 {
   "networkCallRecords" : [ {
     "Method" : "POST",
-    "Uri" : "https://REDACTED.azure-api.net/text/analytics/v3.1-preview.3//entities/linking?showStats=true&stringIndexType=Utf16CodeUnit",
```

Comment:
I'm not sure why the files got renamed with suffix `[1]`. Is it intended? and what does it mean?

Reply:
With the new way to record the JSON output, it is automatically generated. I think it is intended but I forget what it means. I will get back to you once I know the answer. This suffix [1] is all over the places in the SDKs.  such as , https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/src/test/resources/session-records/FormRecognizerAsyncClientTest.recognizeContentAppearance%5B1%5D.json

Reply:
I guess what makes me wonder is you didn't touch this test right? you just disabled the pagination tests... then why this file got modified?

Reply:
it is no longer a case now.